bigtest/network/factories/token.js

```diff
@@ -0,0 +1,24 @@
+import { Factory, faker } from '@bigtest/mirage';
+
+export default Factory.extend({
+  factName: () => faker.random.arrayElement([
+    '[[mysiteid]]',
+    '[[yoursited]]]',
```

Comment:
a very minot nitpick here: probably you meant this to be `yoursiteid`?

Reply:
Yes - thanks for catching!